web/src/datavzrd.js

```diff
@@ -146,7 +146,7 @@ function renderDetailTickBarPlot(value, div, specs, title) {
 }
 
 function colorizeColumn(ah, columns, heatmap, detail_mode, header_label_length) {
-    let index = columns.indexOf(heatmap.title) + 1;
+    let index = columns.indexOf(heatmap.title) + 2;
```

Comment:
I'd rather have a function `getIndex(name: str) -> int` which does this.

Reply:
Done!